src/bundle.rs

```diff
@@ -0,0 +1,17 @@
+use alloy::rpc::types::mev::EthSendBundle;
+use serde::{Deserialize, Serialize};
+
+use crate::SignedOrder;
+
+/// Wraps a flashbots style EthSendBundle with host fills to make a Signet compatible bundle
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct SignetEthBundle {
```

Comment:
let's ensure everything's called `Zenith*` for consistency
```suggestion
pub struct ZenithEthBundle {
```